https://bugs.gentoo.org/557300

fix from upstream

From 2d792707c7ad60c598616183bd1e7c41382961c3 Mon Sep 17 00:00:00 2001
From: Michael Cronenworth <mike@cchtml.com>
Date: Tue, 19 May 2015 16:43:36 -0500
Subject: [PATCH] Always declare gnu89 inline standard

GCC5 switches the minimum inline standard requirements from gnu89 to gnu11.
---
 xbmc/screensavers/rsxs-0.9/lib/argp-fmtstream.h | 4 ----
 xbmc/screensavers/rsxs-0.9/lib/argp.h           | 2 +-
 2 files changed, 1 insertion(+), 5 deletions(-)

diff --git a/xbmc/screensavers/rsxs-0.9/lib/argp-fmtstream.h b/xbmc/screensavers/rsxs-0.9/lib/argp-fmtstream.h
index ff4f575..92eda3b 100644
--- a/xbmc/screensavers/rsxs-0.9/lib/argp-fmtstream.h
+++ b/xbmc/screensavers/rsxs-0.9/lib/argp-fmtstream.h
@@ -198,11 +198,7 @@ extern int __argp_fmtstream_ensure (argp_fmtstream_t __fs, size_t __amount);
 #endif
 
 #ifndef ARGP_FS_EI
-#ifdef __clang__
 #define ARGP_FS_EI extern inline __attribute__ ((__gnu_inline__))
-#else
-#define ARGP_FS_EI extern inline
-#endif
 #endif
 
 ARGP_FS_EI size_t
diff --git a/xbmc/screensavers/rsxs-0.9/lib/argp.h b/xbmc/screensavers/rsxs-0.9/lib/argp.h
index ae7fa04..c83c75e 100644
--- a/xbmc/screensavers/rsxs-0.9/lib/argp.h
+++ b/xbmc/screensavers/rsxs-0.9/lib/argp.h
@@ -559,7 +559,7 @@ extern void *__argp_input (const struct argp *__restrict __argp,
 # endif
 
 # ifndef ARGP_EI
-#  define ARGP_EI extern __inline__
+#  define ARGP_EI extern __inline__ __attribute__ ((gnu_inline)) 
 # endif
 
 ARGP_EI void
-- 
2.4.4

